<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileInfo</class>
 <widget class="QWidget" name="FileInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileInfo</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>106</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>文件名:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>文件大小:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>创建时间:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>最后修改时间:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>最后访问时间:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>462</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="isDirChekBox">
      <property name="text">
       <string>目录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="isFileCheckBox">
      <property name="text">
       <string>文件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="isSymLinkCheckBox">
      <property name="text">
       <string>符号链接</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="isHiddenCheckBox_2">
      <property name="text">
       <string>隐藏</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="isReadableCheckBox">
      <property name="text">
       <string>读</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="isWriteableCheckBox">
      <property name="text">
       <string>写</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="isExecutableCheckBox">
      <property name="text">
       <string>执行</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="browseFileBtn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>。。。</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="fileNameLineEdit_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="fileSizeLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="fileCreateLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="fileModifyLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="fileReadLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>属性:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
